proxygen_cli/cli/command_spec.py

```diff
@@ -39,12 +41,12 @@ def spec(ctx, api, uat):
     "--no-confirm",
     is_flag=True,
     show_default=True,
-    help="Do not prompt for confirmation.",
+    help=HELP_NO_CONFIRM,
 )
 @click.option(
     "--uat",
```

Comment:
would this be taking DRY too far?
```
UAT_KWARGS = {"default":False,"help": HELP_UAT, "is_flag": True}
[...]
@click.option("--uat", **UAT_KWARGS)
```

Reply:
It looks good, yeah. I've nicked it. Also, applied the same to the "no confirm" stuff. I think it sets a good example - the APIM codebase under-uses constants IMO.